client/src/js/app.js

```diff
@@ -746,11 +746,17 @@
     });
 
     $translateProvider.useSanitizeValueStrategy('escape');
-
-    //TODO Try and assign the previous sessions language key here
+  
     $translateProvider.preferredLanguage('fr');
   }
 
+  function localeConfig(tmhDynamicLocaleProvider) { 
+
+    // TODO Hardcoded default translation/ localisation
+    tmhDynamicLocaleProvider.localeLocationPattern('/i18n/locale/angular-locale_{{locale}}.js');
+    tmhDynamicLocaleProvider.defaultLocale('fr-fr');
```

Comment:
True, true.  It was just an observation.

We should research a proper way to bootstrap the app.  The naive (probably easiest) way would be at build time with something like `gulp build --production --locales=en-US,fr-FR,fr-BE`.  I've heard some things about Angular2's router being about to lazy load routes, so maybe we can expect more support for bootstrapping dynamically within Angular.  Of course, we could try to bootstrap before Angular is even loaded, that idea is probably the worst in terms of clarity, maintainability, and portability.